templates/tfengine/recipes/cicd.hcl

```diff
@@ -21,6 +21,25 @@ schema = {
     "scheduler_region",
   ]
   properties = {
+    billing_account = {
+      description = "ID of billing account to attach to this project."
+      type        = "string"
+    }
```

Comment:
why is this added now?

Reply:
It wasn't there but it wasn't needed to afaik.

I added it because I needed the `description` field.

I think it wasn't added because it is not really required, but is provided to the higher order `data = {}` block at the examples